calico-policies/private-network-isolation/au-syd/allow-private-services-pods.yaml

```diff
@@ -33,6 +33,9 @@ spec:
       - 166.9.52.27/32
       - 166.9.54.27/32
       - 166.9.56.32/32
+      - 166.9.244.121/32
+      - 166.9.244.153/32
+      - 166.9.244.185/32
       ports:
```

Comment:
Please also update the list here: https://github.com/IBM-Cloud/kube-samples/blob/master/calico-policies/private-network-isolation/au-syd/allow-private-services.yaml#L32

Reply:
Updated.